app/src/androidMain/kotlin/de/westnordost/streetcomplete/ui/common/image_select/RadioImageList.kt

```diff
@@ -0,0 +1,65 @@
+package de.westnordost.streetcomplete.ui.common.image_select
+
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.key
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.tooling.preview.Preview
+import de.westnordost.streetcomplete.quests.bicycle_repair_station.BicycleRepairStationService
+import de.westnordost.streetcomplete.quests.bicycle_repair_station.asItem
+import de.westnordost.streetcomplete.view.image_select.DisplayItem
+
+@Composable
+fun <T> RadioImageList(items: List<DisplayItem<T>>,
+    onSelect: (List<DisplayItem<T>>) -> Unit,
+    modifier: Modifier = Modifier,
+    itemsPerRow: Int = 4,
+    itemContent: @Composable (item: ImageListItem<T>, index: Int, onClick: () -> Unit, role: Role) -> Unit) {
+    var listItems by remember { mutableStateOf(items.map { ImageListItem(it, false) }) }
+    LaunchedEffect(items) {
+        listItems = items.map { ImageListItem(it, false) }
+    }
```

Comment:
Why would you need this, you already have this in the remember. (Also, the remember should be on the `items`, i.e. `var listItems by remember(items) { ... `

Reply:
It is one of the things where if I remove it, The orientation sensitive items no longer work

Reply:
Seems not using an extra composable also did that trick though